spec/controllers/challenge_claims_controller_spec.rb

```diff
@@ -25,6 +25,59 @@
       it_redirects_to_with_error(root_path, \
                                  "You aren't allowed to see that user's claims.")
     end
+
+    context "for a prompt meme" do
+      let(:signup) { create(:prompt_meme_signup) }
+      let!(:claim_by_other_user) { create(:challenge_claim, collection: collection, claiming_user: create(:user)) }
+
+      it "does not allow a logged in user to access the page with everyone's claims" do
+        fake_login_known_user(user)
+
+        get :index, params: { collection_id: collection.name }
+
+        it_redirects_to_with_error(collection, "Sorry, you're not allowed to do that.")
+      end
+
+      it "does not allow a claiming user to access the page with everyone's claims" do
+        claim.update!(claiming_user: user)
+        fake_login_known_user(user)
+
+        get :index, params: { collection_id: collection.name }
+
+        it_redirects_to_with_error(collection, "Sorry, you're not allowed to do that.")
+      end
+
+      it "allows a claiming user to see their own claims" do
+        claim.update!(claiming_user: user)
+        fake_login_known_user(user)
+
+        get :index, params: { collection_id: collection.name, for_user: true }
+
+        claims = assigns(:claims)
```

Comment:
@sarken, from a thread deleted by Hound:

> I think there's definite value to the tests and it's worth keeping the gem if there's no new equivalent being introduced. However, I have no idea if there is/is not an equivalent coming.

There isn't unfortunately. The Rails team explained their reasons in `https://github.com/rails/rails/issues/18950`, they wanted people to avoid checking instance variables (considered a controller's internals) and start checking the view output more.

We have a lot of tests using `assigns` though, so we'll be sticking with the rails-controller-testing gem for a while. For now and for this PR, this test is fine.